### DISCUSSION TITLE
Proposta

### DISCUSSION THREAD

**Discussion by Lucaccc** [iisgiua/giuaschool#98] [Idee e proposte]
È possibile inserire, al posto della casella di testo “Attività” (durante la firma), un’elenco a discesa, in cui vi sono tutta una serie di attività possibili…come, ad esempio: Lezione, Verifica Scritta, Interrogazione, Interrogazione e Spiegazione, Didattica a Distanza, e così via….

**Comment by trinko** [comment]
La inserisco nelle funzionalità richieste.
